bloom-housing/bloom
# feat: optional app export as spreadsheet

This PR addresses https://github.com/metrotranscom/doorway/issues/753

## Description
This creates a shared export function for both lottery and application export as a spreadsheet

## How Can This Be Tested/Reviewed?
You'll need to add the new partner env variable and turn it to TRUE then request an application export

you should get it in a zip and inside that zip should be a spreadsheet

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
